Shared.IntegrationTests/AdminApi/UsageReportTest.cs

```diff
@@ -29,6 +29,16 @@ public async Task TestUsageAsync()
 
         private void AssertUsageResult(UsageResult result)
         {
+            Assert.True(result.Resources > 0);
+            Assert.True(result.Objects.Used < result.Objects.Limit);
```

Comment:
I would avoid checking used < limit, there can be special cases for certain account types. just check that it is present and of correct type.